frontend/src/main/pages/Ride/RideRequestEditPage.js

```diff
@@ -0,0 +1,75 @@
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import { useParams } from "react-router-dom";
+import RideForm from "main/components/Ride/RideForm";
+import { Navigate } from 'react-router-dom'
+import { useBackend, useBackendMutation } from "main/utils/useBackend";
+
+import { toast } from "react-toastify";
+
+export default function RideEditPage() {
+  let { id } = useParams();
+
+  const { data: ride, _error, _status } =
+    useBackend(
+      // Stryker disable next-line all : don't test internal caching of React Query
+      [`/api/ride_request?id=${id}`],
+      {  // Stryker disable next-line all : GET is the default, so changing this to "" doesn't introduce a bug
+        method: "GET",
+        url: `/api/ride_request`,
+        params: {
+          id
+        }
+      }
+    );
+
+
+  const objectToAxiosPutParams = (ride) => ({
+    url: "/api/ride_request",
+    method: "PUT",
+    params: {
+      id: ride.id,
+    },
+    data: {
+        riderId: ride.riderId,
```

Comment:
Put for backend does not require riderId, automatically fills it in. So, it should not have riderId in params. Also for tests.